spotlight/src/main/java/com/takusemba/spotlight/Spotlight.java

```diff
@@ -126,6 +128,15 @@ public Spotlight setOnSpotlightStartedListener(
     }
 
     /**
+     * listener for target turned up or turned down.
+     */
+    public Spotlight setOnTargetStateChangeListener(
+            @NonNull final OnTargetStateChangeListener listener) {
```

Comment:
I've thought about this, but maybe it's too much work for me.It needs set more than one listener of each target instead of just one listener on Spotlight.